src/ccaligner.h

```diff
@@ -12,13 +12,13 @@
 
 class CCAligner
 {
-    Params * _parameters;
+    Params* _parameters;
```

Comment:
The non-existent style guide says that you need a space between `Params` and `*`. Everywhere else in the code it's like that.